Extension/package.json

```diff
@@ -6540,6 +6543,7 @@
         "node-stream-zip": "^1.15.0",
         "plist": "^3.1.0",
         "posix-getopt": "^1.2.1",
+        "proxyquire": "^2.1.3",
```

Comment:
It looks like this is only needed for tests. @kuchungmsft can you confirm?  I will move it to `devDependencies`.

Reply:
Yes, it's only for testing. Thanks.

Reply:
sinon is also for testing only, thanks.